demos/dropdown/cyclow/es6/src/dropdown.js

```diff
@@ -0,0 +1,24 @@
+import {Block} from 'cyclow'
+
+import {goods} from '../../../goods'
+
+const Dropdown = () => Block({
+  on: {
+    'in.init': () => state => ({goods, chosen: goods[0]}),
+    'dom.select': index => state => ({...state, chosen: goods[index]})
+  },
+  view: ({goods, chosen}) => ({
+    attrs: {className: {vegetable: 'veggie', fruit: 'fruit'}[chosen.type]},
+    content: [
+      'I postit that ',
+      {
+        tag: 'select',
+        on: {change: (e, next) => next({select: e.target.selectedIndex})},
```

Comment:
I tried but it doesn't work. It seems Snabbdom doesn't bind `this` with the node or dom element.